data/weapons.txt

```diff
@@ -403,6 +403,30 @@ outfit "Plasma Cannon"
 		"heat damage" 340
 	description "The Plasma Cannon fires bursts of superheated plasma. In addition to damaging your target, these plasma bursts impart a significant amount of heat, especially to a target whose shields are down."
 
+outfit "Plasma Drill"
+	category "Guns"
+	cost 20000
+	thumbnail "outfit/plasma drill"
+	"mass" 3
+	"outfit space" -3
+	"weapon capacity" -3
+	"gun ports" -1
+	weapon
+		sprite "projectile/plasmab"
+			"frame rate" 4
+		sound "plasmab"
+		"hit effect" "plasma explosion"
+		"inaccuracy" 7
+		"velocity" 2
+		"lifetime" 75
+		"random lifetime" 2
+		"reload" 2
+		"firing energy" .4
+		"firing heat" 1.8
+		"shield damage" .3
+		"hull damage" 3
+	description `The plasma drill is based around the original plasma cannon, scaled down so much though that the range and damage have been greatly reduced. The drill was designed specifically for the Harvester drone, although the demand from solo captains was large enough that the Syndicate released it to be sold individually as well.`
```

Comment:
Gonna take a second to roast the balance of this thing.
*clears throat*

Aside from the strange damage model for a plasma weapon and the fact that it does immense damage for its size (and even more hull DPS than a plasma cannon), this thing produces way too little heat and consumes too little energy. For its current damage model, I'd up the firing energy to at least 0.9, preferable to 1.1. As for the firing heat, I'd up it to at least 3.3 (which even then is generous by my standards).

This thing is CHEAP. I'd at least double the price, but a 50% price hike is good too.

Right now the random velocity from 2 to 4 puts the weapon's range at 150 to 300. For a barrel that long, I think it should be hitting 200 tops. so you could drop the lifetime from 75 to 50, and maybe make its base velocity 3 and random 1 so the range varies from 150 to 200.

Also, no heat damage? Not even a little? It's still a plasma weapon afterall. I'd give it a heat damage of 4.5.